.github/workflows/publish-charts.yaml

```diff
@@ -0,0 +1,89 @@
+name: publish-charts
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository from where we want to build Charts'
+        required: true
+      repo_ref:
+        description: 'The branch/tag that will be used as new version for charts. Usually tag ref'
+        required: true
+      repo_chart:
+        description: 'Object with source path, destination path and chart name'
+        required: true
+      currentChartVersion:
+        description: 'Chart version set in `Chart.yaml`. Default: `LATEST`'
+        required: false
+        default: 'LATEST'
+
+env:
+  REPO: ${{ github.event.inputs.repo }}
+  REPO_REF: ${{ github.event.inputs.repo_ref }}
+  currentChartVersion: ${{ github.event.inputs.currentChartVersion }}
+
+jobs:
+  publish-charts:
+    runs-on: ubuntu-latest
+    name: Publish charts
+    continue-on-error: true
+    strategy:
+      max-parallel: 1
+      matrix: ${{ fromJson(github.event.inputs.repo_chart) }}
+    steps:
+
+      - id: checkout-gh-pages
+        uses: actions/checkout@v2 # Checkout Charts - gh-pages
+        with:
+          ref: 'gh-pages'
+          fetch-depth: 0
+
+      - id: checkout-helm-src
+        uses: actions/checkout@v2 # Checkout Helm source code
+        with:
+          repository: ${{ env.REPO }}
+          ref: ${{ env.REPO_REF }}
+          path: ${{ env.REPO }}
+
+      - id: chart-version
+        name: 'Set version - ${{ env.REPO_REF }}'
+        if: ${{ steps.checkout-gh-pages.outcome == 'success' && steps.checkout-helm-src.outcome == 'success' }}
+        run: |
+          find . \( -name "Chart.yaml" -o -name "values.yaml" \) -exec sed -i s/${{ env.currentChartVersion }}/${{ env.REPO_REF }}/ {} + 
+          echo "Replacing ${{ env.currentChartVersion }} with new version ${{ env.REPO_REF }}."
+        working-directory: ./${{ env.REPO }}
+
+      - id: check
+        name: 'Check if chart already exist'
+        if: ${{ steps.chart-version.outcome == 'success' }}
+        run: |
+          FILE=$(grep name ${{ env.REPO }}/${{ matrix.src_path }}Chart.yaml | awk '{print $2}')
```

Comment:
@ljubon missing slash looks a bit weird to me. not sure if we should rely on the "client" to be aware that they have to send also the end slash as it is in the example:
```
{
  "chart": "armada",
  "src_path": "deployment/armada/",
  "dst_path": "armada/"
}
```

Reply:
@ljubon @mkomatinovic let's add slashes where "missing" in the files we are pushing with this PR. I did some testing, and even if a couple of slashes get asserted, it's going to be fine. Eg: ```grep name myrepo/somechart///Chart.yaml``` will still grep just fine.

Reply:
i've just pushed `/` where i found it it's missing :) Thanks for testing!